### PR TITLE
Fix insecure as true by default, pass flag to client option

### PR DESCRIPTION
**Pull request by fanzhangio**
Quick fix about enable insecure by default. Pass insecure flag to client option

### CODE REVIEW

File: cmd/vcpctl/provision/provision.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ func AddProvision(cmd *cobra.Command) {
 	provisionCmd.Flags().StringVar(&vcport, "port", "", "Specify vCenter Port")
 	provisionCmd.Flags().StringVar(&vcUser, "user", "", "Specify vCenter user")
 	provisionCmd.Flags().StringVar(&vcPassword, "password", "", "Specify vCenter Password")
-	provisionCmd.Flags().BoolVar(&insecure, "insecure", false, "Don't verify the server's certificate chain")
+	provisionCmd.Flags().BoolVar(&insecure, "insecure", true, "Don't verify the server's certificate chain")
```
Comment (frapposelli) [inline cmd/vcpctl/provision/provision.go]: isn't this setting insecure `true` by default?

**Comment by fanzhangio** [comment]
Done. Thanks @fabianofranz @akutz

File: cmd/vcpctl/provision/provision.go (hunk 1 above)
Comment (fanzhangio) [reply cmd/vcpctl/provision/provision.go]: Done. Thanks @fabianofranz @akutz

**Pull request merged**